pkg/controllers/podgroup/pg_controller.go

```diff
@@ -94,6 +97,7 @@ func (pg *pgcontroller) Run(stopCh <-chan struct{}) {
 	cache.WaitForCacheSync(stopCh, pg.podSynced, pg.pgSynced)
 
 	go wait.Until(pg.worker, 0, stopCh)
+	go wait.Until(pg.deletePG, 0, stopCh)
```

Comment:
you need a for-loop here, just like the `worker()` :)
```
func (pg *pgcontroller) worker() {
	for pg.processNextReq() {
	}
}
```

Reply:
@shinytang6 done